pkg/restmapper/controllerrestmapper.go

```diff
@@ -93,9 +93,6 @@ func (m *ControllerRESTMapper) RESTMappings(gk schema.GroupKind, versions ...str
 			return nil, err
 		}
 		group, found := allGroups[gk.Group]
-		if err != nil {
-			return nil, err
-		}
```

Comment:
Already claimed in https://github.com/kubernetes-sigs/kubebuilder-declarative-pattern/blob/bb432bcced715005c6a4d386a6729cb1c9e3a27a/pkg/restmapper/controllerrestmapper.go#L92-L94